tests/lax_numpy_reducers_test.py

```diff
@@ -866,5 +866,19 @@ def testCumulativeSumBool(self):
                              dtype=jnp.bool_)
     np.testing.assert_array_equal(np.array([[True], [True], [False]]), out)
 
+  @jtu.sample_product(dtype=[dtypes.int4, dtypes.uint4, dtypes.int2, dtypes.uint2])
+  @jtu.ignore_warning(category=NumpyComplexWarning)
+  def test_low_bit_sum(self, dtype) -> None:
```

Comment:
Change to camel case to match other test cases in this file.

Reply:
fixed it